web_src/js/features/projects.js

```diff
@@ -59,6 +59,24 @@ export default async function initProject() {
       });
   });
 
+  $('.set-default-project-board').each(function () {
+    $(this).click(function (e) {
+      e.preventDefault();
+
+      $.ajax({
+        url: $(this).data('url'),
+        headers: {
+          'X-Csrf-Token': csrf,
+          'X-Remote': true,
+        },
+        contentType: 'application/json',
+        method: 'POST',
+      }).done(() => {
+        setTimeout(window.location.reload(true), 2000);
```

Comment:
Can the 2s timeout be removed? It seems arbritrary.

Reply:
@silverwind other func in project.js use it too, thats why i use it

I think if you take a look at this JS you will find lots of things to refactor :/